sql/bgw_scheduler.sql

```diff
@@ -73,6 +73,16 @@ AS '@MODULE_PATHNAME@', 'ts_policy_compression_proc'
 LANGUAGE C;
 
 /* continuous aggregates policy */
+CREATE OR REPLACE FUNCTION add_refresh_continuous_aggregate_policy(continuous_aggregate REGCLASS, start_interval "any",  end_interval "any" , schedule_interval INTERVAL, if_not_exists BOOL = false)
+RETURNS INTEGER
```

Comment:
What does the integer return value mean? Is this the common interface for policies?

I would have expected this to return void and throw an error in case of failures.

Reply:
yes, this is the interface for all the policies.